### PR TITLE
Log error messages with context

### PR DESCRIPTION
**Pull request by denalex**

### CODE REVIEW

File: server/pxf-service/src/main/java/org/greenplum/pxf/service/controller/BaseServiceImpl.java [hunk 1]
```diff
@@ -54,42 +56,53 @@ protected BaseServiceImpl(String serviceName,
...
-        // Initialize the configuration for this request
-        Configuration configuration = configurationFactory.
-                initConfiguration(
-                        context.getConfig(),
-                        context.getServerName(),
-                        context.getUser(),
-                        context.getAdditionalConfigProps());
-        context.setConfiguration(configuration);
-
         Instant startTime = Instant.now();
```
Comment (ashuka24) [inline server/pxf-service/src/main/java/org/greenplum/pxf/service/controller/BaseServiceImpl.java:65]: why is the startTime calculated outside the catch if we only use it inside the try?

File: server/pxf-service/src/main/java/org/greenplum/pxf/service/controller/BaseServiceImpl.java [hunk 2]
```diff
@@ -54,42 +56,53 @@ protected BaseServiceImpl(String serviceName,
...
+                        byteCount,
+                        String.format("%.2f", byteRate));
+            } else {
+                log.info("{} completed", context.getId());
+            }
+            return stats;
+        } catch (PxfIOException | PxfRuntimeException | Error e) {
+            log.error(e.getMessage(), e);
+            throw e;
+        } catch (Exception e) {
```
Comment (ashuka24) [inline server/pxf-service/src/main/java/org/greenplum/pxf/service/controller/BaseServiceImpl.java:102]: i noticed that we do this same catch logic in various places. is there a way that we can remove the duplication?

Comment (bradfordb-vmware) [reply server/pxf-service/src/main/java/org/greenplum/pxf/service/controller/BaseServiceImpl.java:104]: We looked at this as a team and have pushed some changes to address this.

**Comment by denalex** [comment]
closing in favor of PR#604 https://github.com/greenplum-db/pxf/pull/604

**Pull request closed without merging**